src/test/java/uk/gov/hmcts/reform/civil/validation/UnavailableDateValidatorTest.java

```diff
@@ -9,6 +9,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.validation.ValidationAutoConfiguration;
 import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.civil.model.RepaymentPlanLRspec;
```

Comment:
The pipeline has failed with checkstyle issues. If this ununsed import one of them. If so, please remove.

Reply:
RepaymentPlanLRspec is not unused, its being used and I have run checkStyle before pushing my code

Reply:
I've checked civil-service pipeline is not broken

Reply:
changes incorporated